app/models/manageiq/providers/nuage/network_manager/event_parser.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers::Nuage::NetworkManager::EventParser
+  def self.event_to_hash(event, ems_id)
+    {
+      :source     => "NUAGE",
+      :event_type => "#{event['entityType']}_#{event['type'].downcase}",
+      :timestamp  => DateTime.strptime((event["eventReceivedTime"] / 1000).to_s, '%s').to_s,
+      :message    => event.to_hash,
```

Comment:
I think `message` should rather be a single string and `full_data` contains the complete hash.